src/main/resources/io/jenkins/plugins/analysis/core/steps/RecordIssuesStep/config.jelly

```diff
@@ -0,0 +1,25 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:i="/issues" xmlns:f="/lib/form">
+
+  <link rel="stylesheet" href="${resURL}/plugin/warnings-ng/css/custom-style.css"/>
```

Comment:
I copied these over straight from `IssuesRecorder` and it looks like they work fine with no changes in the Pipeline step generator. We might be able to do something clever along the lines of:

```
<st:include class="io.jenkins.plugins.analysis.core.steps.IssuesRecorder" page="config.jelly"/>
```

... to avoid duplicating config.jelly, but I couldn't figure out a way to avoid duplicating the help files, and the complexity probably isn't worth it.